docs/conf.py

```diff
@@ -13,14 +13,16 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys
 import os
+import sys
+
+import rhg_compute_tools
```

Comment:
This doesn't work. Sphinx is run from the docs directory, so the `sys.path.insert` below is required before `import rhg_compute_tools`. It's not strict PEP8, but that's probably what's causing your problem.

Reply:
it's probably working on your system because you have rhg_compute_tools already installed I'm guessing?

Reply:
that sounds correct to me! fixed.